### PR TITLE
[8.0][partner_employee_quantity] New module to know the quantity of employees per partner

### PR DESCRIPTION
**Pull request by yajo**
If you need to keep track of the size of your partners by employee number, here's your module. It allows to set a number and a classification.

@rafaelbn @antespi @JavierIniesta.

### CODE REVIEW

File: partner_employee_amount/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Employee amount in partners
+===========================
+
+This module allows to set employee amount for partners, and categorize them by
```
Comment (elicoidal) [inline partner_employee_amount/README.rst]: s/employee amount/number of employees

File: partner_employee_amount/README.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Employee amount in partners
```
Comment (elicoidal) [inline partner_employee_amount/README.rst]: s/employee amount/number of employees

File: partner_employee_amount/README.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* Go to *Sales > Configuration > Address Book > Ranges by amount of employees*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
+* Enable *Is company?*.
+* Go to *Sales & Purchases > Employee amount*.
```
Comment (elicoidal) [inline partner_employee_amount/README.rst]: s/employee amount/number of employees
(etc)

File: partner_employee_amount/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    employee_amount = fields.Integer(
+        oldname="employees_number")
+    employee_amount_range_id = fields.Many2one(
+        "res.partner.employee_amount_range",
+        "Employee amount range",
```
Comment (elicoidal) [inline partner_employee_amount/models/res_partner.py]: s/employee amount/number of employees

**Comment by yajo** [comment]
@elicoidal, what's wrong with "employee amount"? (I'm not English)

**Comment by rafaelbn** [comment]
```
Number of employees
Quantity of employees
quantity of apples
amount of money
amount of milk
number of cars
```

**Comment by elicoidal** [comment]
number is more for discrete quantities (http://www.ecenglish.com/learnenglish/lessons/whats-difference-between-amount-and-number)

**Comment by yajo** [comment]
Thanks, very instructive. I think I will use "quantity" because "employee number" could be interpreted as "assign a number to each employee", which is not the purpose of this module.

**Comment by yajo** [comment]
There it is. Please notice that the :x: in runbot is https://github.com/odoo/odoo/issues/10159.

**Comment by pedrobaeza** [comment]
@Yajo, for the identification number of any employee, the usual term is "Employee ID" or "Employee ID number", so "Number of employees" or "# employees" can't be misunderstood, but anyway, it's no a big deal.

**Comment by yajo** [comment]
Well, it's already done, so if there is no dictionary incompatibility, I'd thank you if you leave it as it is.

**Comment by JavierIniesta** [comment]
Tested in runbot :+1:

**Comment by JMODaERP** [comment]
Tested in runbot :+1:

File: partner_employee_quantity/README.rst [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* Go to *Sales > Configuration > Address Book > Ranges by quantity of employees*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to any partner's form.
+* Enable *Is company?*.
+* Go to *Sales & Purchases > Employee quantity*.
```
Comment (rafaelbn) [inline partner_employee_quantity/README.rst]: This information is not related with sales & purchase, it must be shown in the main form.

**Comment by yajo** [comment]
Fixed.

**Comment by rafaelbn** [comment]
OK, thanks.

Tested in runbot :+1:
